docs/standard/exceptions/best-practices-for-exceptions.md

```diff
@@ -9,20 +9,16 @@ dev_langs:
 helpviewer_keywords: 
   - "exceptions, best practices"
 ms.assetid: f06da765-235b-427a-bfb6-47cd219af539
-author: "mairaw"
-ms.author: "mairaw"
 ---
 # Best practices for exceptions
 
 A well-designed app handles exceptions and errors to prevent app crashes. This section describes best practices for handling and creating exceptions.
 
-## Use try/catch/finally blocks
+## Use try/catch/finally blocks to recover or clean resources
 
-Use `try`/`catch`/`finally` blocks around code that can potentially generate an exception. 
+Use `try`/`catch` blocks around code that can potentially generate an exception ***and*** your code can recover from that exception. In `catch` blocks, always order exceptions from the most specific to the least specific. When your code cannot recover from an exception, don't catch that exception. Enable methods further up the call stack to recover if possible.
```

Comment:
most specific to the least specific: "specific" is unclear. For example, is FileNotFoundException more or less specific that OverflowException? I think that it's better to say "most derived to least derived", and to add a brief explanation (all exceptions are derived from Exception, and the more derived exceptions are not handled if they're preceded by base class exceptions).